apps/camera/locales/camera.en-US.properties

```diff
@@ -91,13 +91,20 @@ preview = Preview
 # screen.
 close-menu-button.ariaLabel = Close menu
 back-button.ariaLabel = Back
+# LOCALIZATION NOTE(setting-option): Used to label an atomic setting-option
```

Comment:
I'd use the full key names in the comments (e.g. ```setting-option.ariaLabel```). Maybe instead of "It is reused" I'd say "The localized value is taken from the setting-option text content.", just to make clear that the value will be localized.